packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -317,14 +317,23 @@ function FontCollection( { slug } ) {
 function PaginationFooter( { page, totalPages, setPage } ) {
 	return (
 		<Flex justify="center">
-			<Button onClick={ () => setPage( 1 ) } disabled={ page === 1 }>
-				&lt;&lt;
+			<Button
+				aria-label={ __( 'first page' ) }
```

Comment:
It would be better to use `label`, which is an API of the `Button` component, instead of `aria-label`.

Reply:
Changed buttons to use label.  Kept select component using `aria-label` so that the label wasn't added above the control.